### PR TITLE
SHS-5796: Remove colorful's half-background link style, use trad's style instead

### PR DESCRIPTION
**Pull request by mariannuar**
# READY FOR REVIEW

## Summary
Remove colorful's half-background link style and use traditional style instead

## Need Review By (Date)
11/05

## Urgency
medium

## Steps to Test
1. Visit https://hs-colorful-ycq1d7e1inzqa0ydu1lcdisx0hwtspbq.tugboatqa.com/components/text-area-typography
2. Hover any kind of link and confirm that they don't have the half-background style anymore. They should look similar to the ones in the traditional theme

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)

### CODE REVIEW

**Comment by ahughes3** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Links:
- https://pr1671-ycq1d7e1inzqa0ydu1lcdisx0hwtspbq.tugboatqa.com
- https://hs-colorful-ycq1d7e1inzqa0ydu1lcdisx0hwtspbq.tugboatqa.com
- https://hs-traditional-ycq1d7e1inzqa0ydu1lcdisx0hwtspbq.tugboatqa.com
- https://english-ycq1d7e1inzqa0ydu1lcdisx0hwtspbq.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67216b5a3e2dd2b98c32436e

Visual Diffs:
- https://dashboard.tugboatqa.com/67216b5a3e2dd2b98c32436e/visualdiffs

File: docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.links.scss [hunk 1]
```diff
@@ -69,24 +69,17 @@
 
 // Link style for main content that is not a heading
 // The default $icon-width is set for the width of external link icons
-@mixin hb-link--inline($icon: false, $icon-width: 16px, $icon-position: right) {
+@mixin hb-link--inline {
   // no font size by default
   color: var(--palette--tertiary);
 
   @include hb-themes(('colorful', 'airy')) {
-    @include hb-link-background-image($icon, $icon-width, $icon-position);
```
Comment (cienvaras) [inline docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.links.scss:77]: I'm pretty sure you're removing all uses of this mixin, you can remove altogether.

**Review by cienvaras — CHANGES_REQUESTED** [review]
@mariannuar Looks good, but I think you're missing some links, for example the card titles:

Screenshot 2024-11-01 at 5 29 34 PM

Please do a general inspection on all components, to confirm that all links with the half-background have been fixed and that there are no undesired effects.

**Comment by ahughes3** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Links:
- https://pr1671-ycq1d7e1inzqa0ydu1lcdisx0hwtspbq.tugboatqa.com
- https://hs-colorful-ycq1d7e1inzqa0ydu1lcdisx0hwtspbq.tugboatqa.com
- https://hs-traditional-ycq1d7e1inzqa0ydu1lcdisx0hwtspbq.tugboatqa.com
- https://english-ycq1d7e1inzqa0ydu1lcdisx0hwtspbq.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67216b5a3e2dd2b98c32436e

Visual Diffs:
- https://dashboard.tugboatqa.com/67216b5a3e2dd2b98c32436e/visualdiffs

**Comment by mariannuar** [comment]
@cienvaras Ready for review again. Please go to https://hs-colorful-ycq1d7e1inzqa0ydu1lcdisx0hwtspbq.tugboatqa.com/components/collections/collections-postcards/vertical-cards and https://hs-colorful-ycq1d7e1inzqa0ydu1lcdisx0hwtspbq.tugboatqa.com/default-views/news

**Review by ahughes3 — APPROVED** [review]
Looks good on our end!

**Comment by cienvaras** [comment]
@ahughes3 LGTM too! I'll assign it to Joe.

**Review by cienvaras — APPROVED** [review]
@mariannuar Looks great, thanks!

**Review by joegl — APPROVED** [review]

**Pull request merged**